docs/dev/error-messages.md

```diff
@@ -385,6 +385,23 @@ If you're already using storage, check to make sure that:
 - You're using the exact name you provided via the rest API, not the original filename. For example, if you uploaded a file named `my_app.apk` to `https://saucelabs.com/rest/v1/storage/YOUR_USERNAME/new_app_name.apk`, your file is available as `storage:filename=new_app_name.apk`.
 - Check the access permissions for the file before retrying. If you have confirmed you have permissions to access the file and you continue to get this error, contact our [Sauce Labs Support](https://saucelabs.com/training-support).
 
+### Rate limit exceeded
+
+**Description**
+
+We do not allow more than ten concurrent Appium commands per Appium session. This limit prevents an overload of the  Appium-server. Note that even though Appium itself defines asynchronous commands, the appium protocol is fundamentally a sequential protocol. The Appium server will never execute commands in parallel, instead it queues all commands it receives and processes them one at a time.
```

Comment:
Not sure where, but it'd be nice to mention somewhere the error message that is returned:

> Rate limit exceeded: you may only run 10 concurrent appium commands per session